modules/updating-virtual-hardware-on-vsphere.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// updating/updating-hardware-on-nodes-running-in-vsphere.adoc
+
+[id="updating-virtual-hardware-on-vsphere_{context}"]
+= Updating virtual hardware on vSphere
+
+To update the hardware of your virtual machines (VMs) on VMware vSphere, you must power off all of your cluster nodes before upgrading. Once your nodes are updated, you can power them back on.
+
+.Prerequisites
+
+* You have cluster administrator permissions.
+* You have a cluster running on vSphere.
+* You have link:https://docs.vmware.com/en/VMware-Telco-Cloud-Operations/1.0/deployment-guide-10/GUID-5249E662-D792-4A1A-93E6-CF331552364C.html[installed `govc`], the CLI for performing administrative actions in vCenter or vSphere.
+
+. Power off all of your cluster nodes in vSphere:
++
+[source,terminal]
+----
+$ govc find <vcenter_vm_folder_path> \ <1>
+  -type m -runtime.powerState poweredOn | xargs -L 1 govc vm.power -off $1
```

Comment:
I could be wrong but does this use vmware tools to shutdown the guest os? If not this is like "pulling the plug"